backend/app/tests/test_organization_foundations.py

```
@@ -129,6 +133,116 @@ def test_list_organization_summaries_materializes_account_state(db_session: Sess
+                workspace_id=workspace.id,
+                name="existing",
+                key_hash="hash",
+                key_prefix="dmarq_existing",
+                scopes="reports:read",
+                active=True,
+            ),
+            WebhookEndpoint(
+                workspace_id=workspace.id,
+                name="hook",
+                url="https://hooks.example/dmarq",
+                secret="stored",
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to argument: "secret"

(S106)